.github/workflows/node.yml

```diff
@@ -0,0 +1,29 @@
+name: Node CI
+on: [push, pull_request]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version:
+          - 10.x
+        npm-script:
+          - test:js
+          - test:types
+          - verify
```

Comment:
Don't run `verify`. Verify just runs all these other steps that you're calling individually, so there's no point in doing that, particularly if Travis is also running the complete verify script.